### DISCUSSION TITLE
Problem with horizontal scroll element in rtl mode

### DISCUSSION THREAD

**Discussion by ashkankiani** [icflorescu/mantine-datatable#576] [Q&A]
Problem with horizontal scroll element in rtl mode. scroll not work
![320760463-1b8a8c66-280f-492d-ae13-39ba55407f8a](https://github.com/icflorescu/mantine-datatable/assets/24645449/86a4155a-59bc-4667-980d-69c9981409d5)

**Comment by icflorescu** [comment]
Open-source projects are created, maintained and made better by people like [Reem Assaf](https://github.com/ReemX), [whose contribution](https://github.com/icflorescu/mantine-datatable/pull/781) finally added RTL support to [Mantine DataTable](https://icflorescu.github.io/mantine-datatable/)! 🎉🙏
